jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -26,8 +26,10 @@
 class OutlierDetectionStep(Step):
     """Flag outlier bad pixels and cosmic rays in DQ array of each input image.
 
-    Input images can be listed in an input association file or already opened
-    with a ModelContainer.  DQ arrays are modified in place.
+    Input images can be listed in an input association file or dictionary,
+    or already opened with a ModelContainer or ModelLibrary.
+    DQ arrays are modified in place.
+    SCI, ERR, and VAR arrays are updated with NaN values matching the DQ flags.
```

Comment:
This is my fault for being vague in my previous comments, but there is no actual VAR array - it is the VAR_POISSON, VAR_RNOISE, and VAR_FLAT arrays that are updated.  Users will probably know what you mean, but we should probably either spell out the extension names, or else call them "variance arrays", rather than "VAR".

Reply:
Thanks, yeah I know there are three separate arrays, but you're right I should probably spell them out here and everywhere I copied that